packages/app-root/README.md

```diff
@@ -10,11 +10,33 @@ This package should be cloned as part of the [front-end-monorepo](https://github
 
 Starts a development server on port 3000.
 
+In addition, you must have the local sub-domain [setup](https://stackoverflow.com/c/zooniverse/questions/109) to get past CORS errors when authenticating with Panoptes in your hosts file.
+
+Once you have the hosts file configured, you'll be able to use one of those subdomains to do local development for projects on and be able to authenticate with Panoptes, i.e. at `https://local.zooniverse.org:3000/users` or `https://localhost.zooniverse.org:3000/users`.
+
+### Docker
+
+- `docker-compose up -d` to run a dev server, in the background, on http://localhost:3000 using `yarn dev`. The `--build` flag can be used to build the container. This builds and runs a local image which matches the GitHub Action build except for running behind a proxy. Note: `devcert` is not yet setup for our docker build for local development.
+- `docker-compose down` to stop the dev containers.
+<!-- - `docker-compose run --rm root test` to run the tests. -->
+
 ### Node
+
 ```sh
 yarn dev
 ```
-or
+
+## Running in production
+
+[NextJS Production Checklist](https://nextjs.org/docs/app/building-your-application/deploying/production-checklist)
+
+The production server is started on port 3000 by default.
+
+### Docker
+
+- `docker-compose run --rm root start` to run a webpack production build on http://localhost:3000. The `--build` flag can be used to build the container. This builds and runs a local image which matches the GitHub Action build except for running behind a proxy.
```

Comment:
I could not get this to work. But I could also not get it to work with `app-project`, so I'm not sure if it's generally not working, something I'm doing wrong, or something unique to the `app-root` setup. If it's generally not working, I should remove this section from the README until it is working again.